src/controllers/users.controller.js

```diff
@@ -1,7 +1,8 @@
 import lodash from 'lodash';
-import mailgun from 'mailgun-js';
+// import mailgun from 'mailgun-js';
```

Comment:
@giranezafiacre Nice work, the feature is working perfectly for me. Things to fix would be to remove the commented out line if it is not needed and some es lint errors in `/src/helper/templates/forgotPassword/`